src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+    private final Map<LottoRank, Integer> result;
+
+    public  LottoResult(Map<LottoRank, Integer> result) {
+        this.result = result;
+    }
+
+    public int calTotal() {
```

Comment:
```suggestion
    public Money calTotal() {
```
Money 포장했는데 적극 활용해 본다면